tests/lib/Content/Form/Provider/GroupedContentFormFieldsProviderTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\ContentForms\Content\Form\Provider;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\TextLine\Value;
+use eZ\Publish\Core\Helper\FieldsGroups\FieldsGroupsList;
+use eZ\Publish\Core\Repository\Values\ContentType\FieldDefinition;
+use EzSystems\EzPlatformContentForms\Data\Content\FieldData;
+use Ibexa\ContentForms\Content\Form\Provider\GroupedContentFormFieldsProvider;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormInterface;
+
+final class GroupedContentFormFieldsProviderTest extends TestCase
+{
+    public function testGetGroupedFields(): void
+    {
+        $fieldsGroupsListMock = $this->createMock(FieldsGroupsList::class);
+        $fieldsGroupsListMock
+            ->expects($this->exactly(3))
+            ->method('getFieldGroup')
+            ->withConsecutive()
+            ->willReturnOnConsecutiveCalls('group_1', 'group_2', 'group_2');
+
+        $fieldsGroupsListMock
+            ->expects($this->once())
+            ->method('getGroups')
+            ->withAnyParameters()
```

Comment:
It's a no-op. No parameter check is the default.

```suggestion
```

Reply:
fixed